jpa/eclipselink/src/test/java/example/springdata/jpa/eclipselink/SpringDataJpaEclipseLinkApplicationTests.java

```diff
@@ -0,0 +1,259 @@
+package example.springdata.jpa.eclipselink;
+
+import example.springdata.jpa.eclipselink.domain.Gender;
+import example.springdata.jpa.eclipselink.domain.Participant;
+import example.springdata.jpa.eclipselink.domain.Race;
+import example.springdata.jpa.eclipselink.repositories.ParticipantRepository;
+import example.springdata.jpa.eclipselink.repositories.RaceRepository;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.SpringApplicationConfiguration;
+import org.springframework.test.context.junit4.SpringJUnit4ClassRunner;
+
+import java.text.SimpleDateFormat;
+import java.util.Date;
+import java.util.List;
+
+import static org.hamcrest.Matchers.is;
+import static org.junit.Assert.*;
+
+@RunWith(SpringJUnit4ClassRunner.class)
+@SpringApplicationConfiguration(classes = SpringDataJpaEclipseLinkApplication.class)
+public class SpringDataJpaEclipseLinkApplicationTests {
```

Comment:
Simplify, simplify, simply. Remember, this should become an exhaustive integration test for CRUD methods on EclipseLink. Rest assured we do this in Spring Data JPA itself. The example should focus on a single aspect, which in this particular case is the general setup of a Spring Data JPA project with EclipseLink. The execution of a query method alongside a CRUD method should be entirely sufficient. Everything else just distracts from the core aspect and needs to be maintained going forward.